tests/system/api/test_artifacts.py

```diff
@@ -58,3 +61,73 @@ def test_fail_overflowing_artifact(self):
         assert (
             "Failed committing changes to DB" in run["status"]["error"]
         ), "run should fail with a reason"
+
+    def test_model_artifact_tags(self):
```

Comment:
add enterprise mark (v3io)

Reply:
possibly move to unit test and log the artifact locally :)

Reply:
And please explain the test flow

Reply:
> possibly move to unit test and log the artifact locally :)

Couldnt agree more! worth unit-testing as well

Reply:
@alonmr @liranbg I deleted the system test and replaced it with a simple unit test that actually checks what this PR does :|